keras_cv/layers/preprocessing/random_resized_crop.py

```diff
@@ -0,0 +1,91 @@
+from keras import backend
```

Comment:
lets also add a script to examples showing the outputs of this layer.  Lets base it on the color degeneration example:


https://github.com/keras-team/keras-cv/blob/master/examples/layers/preprocessing/random_color_degeneration_demo.py